Misc/NEWS

```diff
@@ -42,6 +42,10 @@ Extension Modules
 Library
 -------
 
+- bpo-30363: Running Python with the -3 option now warns about regular
+  expression syntax that is invalid or has different semantic in Python 3
+  or will change the behavior in future Python versions.
```

Comment:
This doesn't change the behavior. Just warnings are raised for suspicious regexpes in py3k compatible mode.

It is easy to write code working on Python 2 and 3 that doesn't emit a warning. In case of a bad escape, just remove a redundant backslash if the code is correct. But it is likely that the warning points to a bug (@jwilk have found a number of such bugs in third-party projects). If you use `re.split()` with a pattern always matching an empty string (e.g. `r'\b'`), it will never worked, this is a bug. If you use `re.split()` with a pattern that can match an empty string (e.g. `r'\s*'`), you should change it to a pattern that can't match an empty string (`r'\s+'`) for avoiding a warning.